src/device/usbd.c

```diff
@@ -801,7 +801,7 @@ static bool process_get_descriptor(uint8_t rhport, tusb_control_request_t const
     case TUSB_DESC_CONFIGURATION:
     {
       tusb_desc_configuration_t const* desc_config = (tusb_desc_configuration_t const*) tud_descriptor_configuration_cb(desc_index);
-      TU_ASSERT(desc_config);
+      TU_VERIFY(desc_config);
```

Comment:
This should be assert to tell application in the log that they are returning NULL config descriptors. Again it is unlikely host request the out-of-bound config descriptor. It is likely the application logic issue.